### ISSUE TITLE
Frontend generates the filled Excel file from that JSON Schema

### ISSUE THREAD

**Issue by geekygirlsarah** [GSA-TTS/FAC#483]
Breakout issue of #443
~Blocked on #478~
Blocked on #481, #482

### Background

The FAC has to generate an Excel file for the user to download to fill in with their federal awards. This task is to generate the filled Excel file with validation that matches the data from the API endpoint.

### Acceptance Criteria (We'll know we're done when...)

- [ ] Generates an Excel file with API-fetched data in it
- [ ] Sends file to user
- [ ] Test to ensure file is downloaded
   - Don't test stuff in the file yet, this will take a while until the Excel uploading features are done

### Tasks

- [ ] Parse data from API endpoint (since getting empty data is already done)
- [ ] Modify Excel generation so it fills in data into the spreadsheet
- [ ] Send the file to the user
- [ ] Manually test it (check validations work)
- [ ] Write tests (if they don't exist) to check the file is downloaded

**Comment by jadudm** [comment]
We are not implementing the XLSX download at this time. It is a post-MVP issue to discuss and implement how we will support users with their data. Also, the API provides *far more functionality* than the XLSX download.

All of this is to say that there is more work with users before we make decisions here, so I'm closing this as "not planned."
